src/MyElefant.php

```diff
@@ -270,7 +290,26 @@ public function sendSms($campaignId, $content)
                 );
                 return true;
             }
+        } else if ($response->getStatusCode() == 401) {
+            $body     = $response->getBody();
+            $arr_body = json_decode($body);
+            if ($arr_body->errors->Authorization[0] === "Invalid access token" && $this->tokenFlag === 0) {
+                $this->token = $this->setAuthentification($this->secretKey);
+                $this->tokenFlag = 1;
+                $this->guzzleCatcher = true;
+                $this->setLog(
+                    'info',
+                    'Regeneration du token'
```

Comment:
maybe use that const `REGENERATING_TOKEN ` ? (https://github.com/digitregroup/php-myelefant-client/pull/5/files#diff-3f951923c25f71476abe6a473f56ff6fR21)

Reply:
Hum... of course